README.md

```diff
@@ -155,7 +155,7 @@ If you want to run the test suite, make sure you have your environment set up, a
 
 ```
 # run the unit tests
-$ make check
+$ make test-unit
```

Comment:
I think we'd like to update the comment above this.  

`# run the verifiers, unit tests, and and command test`